app/src/main/res/layout/activity_transaction_detail.xml

```diff
@@ -130,27 +130,74 @@
           />
 
       <TextView
-          android:id="@+id/status_label"
+          android:id="@+id/payment_method_label"
           style="@style/TransactionDetailsTextStyle"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
           android:layout_marginStart="@dimen/half_large_margin"
           android:layout_marginTop="35dp"
-          android:text="@string/transaction_status_label"
+          android:text="Payment Method"
+          android:visibility="gone"
           app:layout_constraintStart_toStartOf="parent"
           app:layout_constraintTop_toBottomOf="@id/top_separator"
           />
 
+      <LinearLayout
+          android:id="@+id/payment_method"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginEnd="@dimen/half_large_margin"
+          android:orientation="horizontal"
+          android:visibility="gone"
+          app:layout_constraintBottom_toBottomOf="@id/payment_method_label"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintTop_toTopOf="@id/payment_method_label"
+          >
+
+        <ImageView
+            android:id="@+id/payment_method_icon"
+            android:layout_width="21dp"
+            android:layout_height="21dp"
+            android:layout_marginEnd="@dimen/small_margin"
+            android:adjustViewBounds="true"
+            android:contentDescription="@string/empty"
+            android:scaleType="centerInside"
+            tools:src="@drawable/ic_payment_method_paypal"
```

Comment:
Idk if this is what we talked about, but in the review we noticed that the payment icon was too close to the text